dap-mode.el

```diff
@@ -1021,6 +1021,22 @@ should be started after the :port argument is taken.
                    breakpoints)
           (lsp-workspace-set-metadata "Breakpoints" breakpoints))))))
 
+(defun dap--remove-debug-template (template-name &optional all)
+  "Remove template configurations with template-name out of the registered list.
```

Comment:
As mentioned, we should write function arguments in all caps in docs.

Reply:
Done